lib/Runtime/Library/JsBuiltIn/JsBuiltIn.js

```diff
@@ -34,6 +34,9 @@
     __chakraLibrary.raiseFunctionArgument_NeedFunction = platform.raiseFunctionArgument_NeedFunction;
     __chakraLibrary.functionBind = platform.builtInJavascriptFunctionEntryBind;
     __chakraLibrary.objectDefineProperty = _objectDefineProperty;
+    __chakraLibrary.isNaN = platform.builtInGlobalObjectEntryIsNaN;
```

Comment:
You added JavascriptNumber::IsNaN but are still using GlobalObject::IsNaN, it seems?

Reply:
I'll be removing JavascriptNumber::IsNaN.